### PR TITLE
Add cstrike_stocks.inc

### PR DESCRIPTION
**Pull request by Bara20**
Add CS_GetEntitySpeed + CS_SetEntitySpeed

I don't which games has m_flLaggedMovementValue.

### CODE REVIEW

**Comment by WildCard65** [comment]
Probably should have it match tf2_stocks.inc in that cstrike_stocks should include cstrike

File: plugins/include/cstrike_stocks.inc [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+}
+
+/**
+ * Sets an entity's speed.
+ *
+ * @param entity	Entity index.
+ * @param amount	Speed to set (default = 1.0, half = 0.5, double = 2.0).
+ * @error			Invalid entity index, or lack of mod compliance.
+ */
+stock void SetEntitySpeed(int entity, float amount)
```
Comment (WildCard65) [inline plugins/include/cstrike_stocks.inc]: Missing CS_ prefix?

**Comment by KyleSanderson** [comment]
We appreciate the thought, however this has serious repercussions such as impacting not only movement; but fall speed. If you're able to bypass m_flMaxspeed's restriction (or similar) and have it stick like this; that would absolutely be a welcomed addition. Changing this prop only results in quite unfortunate UX, so I'm closing this.

**Comment by Drifter321** [comment]
Alright, lets start with why are stocks required? AFAIK the prop exists on both cs:go and cs:s, so why is this needed? Also this isnt a great way to set a players speed.

**Pull request closed without merging**